lib/chef/provider/script.rb

```diff
@@ -18,6 +18,7 @@
 
 require "tempfile"
 require "chef/provider/execute"
+require "chef/win32/security" if Chef::Platform.windows?
```

Comment:
@btm, I tried to use `node[:os]` here too, but it raises error that `node object is not defined`.